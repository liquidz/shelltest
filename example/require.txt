bash$ echo hello
hello

@require example/required.txt
