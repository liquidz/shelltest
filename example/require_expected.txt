$ shelltest example/require.txt
..

2 tests, 0 failures
